src/lib/viewers/__tests__/BaseViewer-test.js

```diff
@@ -1189,7 +1192,9 @@ describe('lib/viewers/BaseViewer', () => {
             base.createAnnotator();
 
             expect(base.options.boxAnnotations.getOptions).toBeCalled();
-            expect(base.createAnnotatorOptions).toBeCalledWith(expect.objectContaining(createOptionsArg));
+            expect(base.createAnnotatorOptions).toBeCalledWith(
+                expect.objectContaining({ ...createOptionsArg, initialColor: '#000' }),
```

Comment:
Can we move this to the `createOptionsArg` definition above instead of spreading it again?